ui/src/tool-panel/location-track/location-track-infobox-duplicate-track-entry.tsx

```diff
@@ -136,10 +136,7 @@ function validateIsPartialDuplicate(
     const isPartialDuplicate = duplicate.duplicateStatus.match === 'PARTIAL';
     const overLappingLength = duplicate.duplicateStatus.overlappingLength;
     const nonOverlappingLength =
-        (duplicate.length != undefined &&
-            overLappingLength != undefined &&
-            duplicate.length - overLappingLength) ||
-        undefined;
```

Comment:
Näissä voisi varmaankin käyttää `!==`, (ja miksiköhän tuolla on ollut jo vanhastaan tuo ´|| undefined´, luulisi että tuo on `undefined` ilman sitäkin) 🤔

Reply:
Tuon tyypiksi tulee `number | false` jos tuon `|| undefinedin` ottaa veks, aiemmin ja nyt siis tyypiksi tulee `number | undefined`

Tosin en lukenut eteenpäin että minkä vuoksi, ehkäpä ei haluta booleania mukaan tyyppijoukkoon tms (pituus joka on false/boolean kuulostaakin vähän hassulta)

Reply:
Njoo en itse asiassa edes katsonut mitä nämä tekee, otin vain undefined checkin pois koska se ei voi olla undefined. Muutenkin tämä on vähän jännä että erotus on laitettu tuonne sulkuihin mukaan jolloin ei saada tulokseksi koskaan 0 vaan se mappautuu undefinediksi, mutta näyttäisi olevan tarkoituksella. Kaippa tuon  !== voisi ainakin vaihtaa

Reply:
Eikunjaa, en ehkä halua puuttua tähän senkään vertaa. Tuokin saattaisi olla ihan tarkoituksella jos tuossa halutaan samalla myös 0 pois tuolta vertailusta. Pitäisi testailla tuota vähän paremmin ennen kun uskaltaa tuohon koskea.